scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -138,6 +138,7 @@ class FossId(
         user = fossIdScannerOptions["user"]
             ?: throw IllegalArgumentException("No FossId User configuration found.")
         packageNamespaceFilter = fossIdScannerOptions["packageNamespaceFilter"].orEmpty()
+        log.info { "FossID server URL is $serverUrl." }
```

Comment:
Why does this line go exactly here? Wouldn't it make more sense directly after `serverUrl` is assigned, or directly before `serverUrl` is used for the first time?